app/src/main/java/edu/wpi/first/shuffleboard/app/sources/NetworkTableSourceType.java

```diff
@@ -0,0 +1,18 @@
+package edu.wpi.first.shuffleboard.app.sources;
+
+import edu.wpi.first.shuffleboard.api.sources.SourceType;
+import edu.wpi.first.shuffleboard.api.sources.SourceTypes;
+
+public final class NetworkTableSourceType extends SourceType {
```

Comment:
`DataType` doesn't use annotations. I think this is fine, especially since it'll get changed (again) when we add plugins for #102